webtool/sperre.py

```
@@ -366,7 +425,27 @@ def datei(pfad: str, stale: float = STALTES_ALTER):
+        # einziger solcher Fehler nach dem erzwungenen Griff sofort in "ungeschuetzt weiter",
+        # und der Griff selbst raeumte ein Lock weg, das 10 ms spaeter regulaer frei gewesen
+        # waere.
+        #
+        # **Warum die Obergrenze aus #191 dabei stehen bleibt — die Form des Arguments, denn
+        # sie steht in keiner einzelnen Zeile:** es gibt genau drei Rundenarten. (A) `mkdir`
+        # gelingt → `break`. (B) `FileExistsError` → setzt `hakelig_seit` auf `None` UND faellt
+        # in diese Pruefung durch. (C) anderer `OSError` → bricht nach `_HAKELIG_S` von sich aus
+        # ab und ueberspringt sie. `hakelig_seit` wird ausser bei der Initialisierung **nur** von
+        # (B) geloescht, und (B) erreicht diese Pruefung immer; nach der Frist setzt das erste
+        # (B) `erzwungen`, das zweite bricht ab — `erzwungen` ist ein Einmal-Riegel. Aufgeschoben
+        # wird damit hoechstens **2 × `_HAKELIG_S`** (ein Nachsichtsfenster kann vor dem Griff
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)